specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```
@@ -25,7 +25,7 @@
         "$ref": "#/parameters/Endpoint"
       }
     ]
-  },
+  },  
```

Adding the trailing space doesn't look related
```suggestion
  },
```